packages/ui/Layout/Cell.js

```diff
@@ -109,6 +130,7 @@ const CellBase = kind({
 
 	defaultProps: {
 		component: 'div',
+		grow: false,
```

Comment:
We don't need to specify false default props.

Reply:
I will fix it.